src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import java.io.*;
+import java.util.*;
+import java.util.Date;
 public class Duke {
+    static String line = "    ____________________________________________________________";
+    public static LinkedList<Task> storage = new LinkedList<Task>(); 
     public static void main(String[] args) {
+        
```

Comment:
You might not need to have an extra line here. Instead, consider putting a line before psvm.